services/checks/checks.go

```diff
@@ -273,7 +274,7 @@ func (s *Service) restartChecks(ctx context.Context) {
 }
 
 // GetSecurityCheckResults returns the results of the STT checks that were run. It returns services.ErrSTTDisabled if STT is disabled.
-func (s *Service) GetSecurityCheckResults() ([]check.Result, error) {
+func (s *Service) GetSecurityCheckResults() ([]utils.STTCheckResult, error) {
```

Comment:
what kind of error?

Reply:
can't import package because its blacklisted, something like that, that test is in `main_test.go`

that is why we use `deps.go` approach to call functions of a service that are outside the management package

Reply:
Oh, that sucks 🙁 . I think it's better to place these files in `services` root, instead of `utils`.

Also where `GetCheckResults` is used? UI uses alertmanager API to get check results.

Reply:
yes, but there is a frontend ticket to migrate to `GetCheckResults` which might get picked up in a few days so need to merge this PR first as it is a blocker for the frontend ticket

Reply:
Ok, but replace these files is not a big deal, right?

Reply:
yeah just moved them to services root and tests pass

Reply:
here https://github.com/percona/pmm-managed/pull/777/commits/4d0d335a75e126b8a39a131cbc675e31c796e0e2